lib/supavisor/logger/logflare_formatter.ex

```diff
@@ -26,15 +27,27 @@ defmodule Supavisor.Logger.LogflareFormatter do
       mfa
     ]a
 
+  @default_context_transformations [
+    mfa: &__MODULE__.transform_mfa/1
+  ]
+
   @impl true
   def check_config(_), do: :ok
 
   @impl true
   def format(%{msg: msg, level: level, meta: meta}, opts) do
-    context_keys = [Map.get(opts, :context, []) | @default_context]
+    context_keys = Map.get(opts, :context, []) ++ @default_context
```

Comment:
This was also broken, the keys went outside of `context` because the list wasn't properly merged